api/src/pcapi/sandboxes/scripts/creators/test_cases/__init__.py

```diff
@@ -588,6 +590,77 @@ def _create_allocine_venues() -> list[offerers_models.Venue]:
     return venues
 
 
+def create_offers_interactions() -> None:
```

Comment:
```suggestion
def create_offers_interactions() -> None:
    logger.info("create_offers_interactions")
```
ça nous mettrait bien de commencer à logger ce qu'il se passe dans ce trou noir